CheckWriting2.js

```diff
@@ -0,0 +1,36 @@
+var test = require('mocha').it,
```

Comment:
looks like you pushed some (empty/different) files a few times. Do you need some help with this?